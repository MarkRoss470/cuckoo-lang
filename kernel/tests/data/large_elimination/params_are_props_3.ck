data SomeProp : Prop where
  | c : SomeProp -> SomeProp

-- This `Prop` judgement is wrong, but never reached because the level error is encountered first
def some_prop_rec : Prop := SomeProp.rec