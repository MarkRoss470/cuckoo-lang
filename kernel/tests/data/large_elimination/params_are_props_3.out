ERROR at tests/data/large_elimination/params_are_props_3.ck 5:29-5:41: SomeProp.rec takes 1 level argument(s), but 0 were provided.